### ISSUE TITLE
Scored Quests

### ISSUE THREAD

**Issue by fjbagley** [timberline-secondary/hackerspace#546]
A quest could be worth 5-10XP. A student submits that quest. If they meet the minimum requirements and get approved they get 5XP (pass) or next to the approve button there could be a dropdown with the different number of XP they could receive. 

The extra XP could either be given via currency badges - pennies, nickels and dimes (BD figures that out) or could be recorded along with the quest approval say "+3" for an 8XP.

Would be very useful in non digital media courses. I could see some uses in digital media too.

**Comment by tylerecouture** [comment]
@fjbagley do you think this request has been covered by the quest option that allows students to enter an XP value?

**Comment by fjbagley** [comment]
@tylerecouture No, but I think it is getting close. An admin would need to have a way that after the quest is submitted with XP selected (student self-evaluates), they could approve but then assign a different XP (marked the quest).
